exporter/awsprometheusremotewriteexporter/auth.go

```diff
@@ -57,6 +58,15 @@ func (si *signingRoundTripper) RoundTrip(req *http.Request) (*http.Response, err
 
 	// Clone request to ensure thread safety.
 	req2 := cloneRequest(req)
+
+	// Add the runtime information to the User-Agent header of the request
+	curUA := req2.Header.Get("User-Agent")
+	if len(curUA) > 0 {
+		si.runtimeInfo = curUA + " " + si.runtimeInfo
+	}
```

Comment:
```suggestion
	ua := req2.Header.Get("User-Agent")
	if len(ua) > 0 {
		ua = ua + " " + si.runtimeInfo
	} else {
	  ua = si.runtimeInfo
	}
```

You're going to append to this `si.runtimeInfo` constant every single signing time with this code right now!

Reply:
I realize the mistake, have changed according to your suggestion